Gemfile

```diff
@@ -72,6 +72,7 @@ gem "rubyzip",                        "~>1.2.2",       :require => false
 gem "rugged",                         "~>0.27.0",      :require => false
 gem "snmp",                           "~>1.2.0",       :require => false
 gem "sqlite3",                        "~>1.3.0",       :require => false
+gem "sys-filesystem",                 "~>1.2.0",       :require => false
```

Comment:
I don't know if we can trust the author 🤣

Reply:
;)